controllers/operator/configmap.go

```diff
@@ -272,6 +272,11 @@ func (r *AuthenticationReconciler) handleConfigMap(instance *operatorv1alpha1.Au
 					currentConfigMap.Data["MASTER_HOST"] = newConfigMap.Data["MASTER_HOST"]
 					cmUpdateRequired = true
 				}
+				if _, keyExists := currentConfigMap.Data["MASTER_PATH"]; !keyExists {
```

Comment:
The stanza immediately preceding this one is dealing with MASTER_PATH - why is this one necessary?